### PR TITLE
Fixes console commands being interrupted by errors.

### PR DESCRIPTION
**Pull request by MaxIsJoe**
CL: [Fix] Fixes console commands being interrupted by errors.

### CODE REVIEW

File: UnityProject/Assets/IngameDebugConsole/Scripts/DebugLogUnitystationCommands.cs [hunk 1]
```diff
@@ -29,10 +30,36 @@ public class DebugLogUnitystationCommands : MonoBehaviour
...
+				{
+					report.AppendLine($"Player Script not null: {PlayerManager.LocalPlayerScript != null}");
+					report.AppendLine($"Player Info not null: {PlayerManager.LocalPlayerScript.PlayerInfo != null}");
+					report.AppendLine($"Player connection not null: {PlayerManager.LocalPlayerScript.PlayerInfo?.Connection != null}");
+				}
+				report.AppendLine("-------");
+				report.AppendLine(e.ToString());
+				Logger.LogError(report.ToString());
+				// To avoid disruption, allow IsAdmin() to continue normally; the server will handle anti-cheat in it's own functions.
+				return true;
```
Comment (Bod9001) [inline UnityProject/Assets/IngameDebugConsole/Scripts/DebugLogUnitystationCommands.cs]: shouldn't this return false?

Comment (MaxIsJoe) [reply UnityProject/Assets/IngameDebugConsole/Scripts/DebugLogUnitystationCommands.cs]: Read the comment

Comment (MaxIsJoe) [reply UnityProject/Assets/IngameDebugConsole/Scripts/DebugLogUnitystationCommands.cs]: `allow IsAdmin() to continue normally; the server will handle anti-cheat in it's own functions`

Comment (ThatDan123) [reply UnityProject/Assets/IngameDebugConsole/Scripts/DebugLogUnitystationCommands.cs]: Did you trigger the debug command in the lobby? That would mean LocalPlayerScript would be null, you'd want LocalViewerScript (think thats what it is called for player viewer script) when in lobby.

Comment (MaxIsJoe) [reply UnityProject/Assets/IngameDebugConsole/Scripts/DebugLogUnitystationCommands.cs]: No, while in a round and in a character

**Review by ThatDan123 — APPROVED** [review]

**Pull request merged**